packages/fluentui/react-northstar/src/components/Embed/Embed.tsx

```diff
@@ -241,6 +241,7 @@ Embed.defaultProps = {
   as: 'span',
   accessibility: embedBehavior,
   control: {},
+  variables: {},
```

Comment:
nit: I don't wanna be that guy but since this is a regression, should we add a unit test ?

Reply:
we could :) let me